faker/__init__.py

```diff
@@ -1,3 +1,477 @@
+# from README.rst
```

Comment:
I'd rather not have to main the same Documentation twice. Could you undo this changes?

Reply:
ok
maybe make it shorter?

Reply:
What is your reasoning of adding it to the file at all?

Reply:
????